include/DisplayValue.h

```diff
@@ -15,10 +16,9 @@
 #include <stdlib.h>           /* abs */
 #include <typeinfo>
 
+#include <../lib/SPIBus/SPIBus.h>
```

Comment:
Is there a specific reason that you are not use `<SPIBus.h>` directly? The header should be in the default include path, right?

Reply:
Terribly it is not found in the usual way :(
If you can fix then I am glad!

Reply:
resolve for now, let's try in next round